tarsier/tag_utils.ts

```diff
@@ -150,7 +152,7 @@ function getElementXPath(element: HTMLElement | null) {
     path_parts.unshift(prefix);
     element = element.parentNode as HTMLElement | null;
   }
-  return iframe_str + "//" + path_parts.join("/");
+  return window.fixNamespaces(iframe_str + "//" + path_parts.join("/"));
```

Comment:
if this should only be used to fix specific html tags like sg.someName, can we do this above for tags only?